### DISCUSSION TITLE
ListAction

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#5499] [General]
Hi,

Is there any reason why Filament has CreateAction, View/EditAction, DeleteAction but no ListAction.

So if I am on an edit page, I can jump back to the list using the ListAction on the top of the page.

I know I can click on cancel, but ListAction in my opinion completes the list of actions.

Just wondering....

Thanks

**Comment by danharrin** [comment]
I think its because theres no actual logic we need to reuse in such a button... you can just create a very simple action with a `url()` for that purpose
